cephfs/admin/bytecount.go

```diff
@@ -1,3 +1,4 @@
+//go:build !luminous && !mimic
```

Comment:
Since luminous and mimic have not been supported for a couple of releases would it make sense to first remove the old build constraints prior to running `gofmt` and having it add a bunch of very-unlikely-to-be-useful constraints in this new format?